mod/01_rating/lib/stylesheets/style_drag_and_drop.scss

```diff
@@ -332,9 +332,15 @@ $bright-text-color: #e5e5e5;
       color:$dark-text-color;
 
     }
-    >.data {
-      width: auto!important;
+    >div{
+      >.data {
+        width: auto!important;
```

Comment:
essentially, yes.  "bang" is a short way to say "exclamation point".

my question was more about stylistic convention than functionality; I know that it works, but I don't know whether we'd generally prefer to have a space there (before minifying)